lib/ts/recipe/emailverification/components/features/emailVerification/index.tsx

```diff
@@ -142,25 +143,27 @@ class EmailVerification extends PureComponent<Prop, { status: "READY" | "LOADING
         };
 
         return (
-            <ComponentOverrideContext.Provider value={componentOverrides}>
-                <FeatureWrapper
-                    useShadowDom={this.props.recipe.config.useShadowDom}
-                    defaultStore={defaultTranslationsEmailVerification}>
-                    <Fragment>
-                        {/* No custom theme, use default. */}
-                        {this.props.children === undefined && <EmailVerificationTheme {...props} />}
-                        {/* Otherwise, custom theme is provided, propagate props. */}
-                        {this.props.children &&
-                            React.Children.map(this.props.children, (child) => {
-                                if (React.isValidElement(child)) {
-                                    return React.cloneElement(child, props);
-                                }
-
-                                return child;
-                            })}
-                    </Fragment>
-                </FeatureWrapper>
-            </ComponentOverrideContext.Provider>
+            <UserContextWrapper userContext={this.props.userContext}>
```

Comment:
Why do we add the wrapping in the feature component? In other feature components the wrapping is added via the `getFeature` function correct?

Reply:
You are right, this was redundant. Removed